polkadot/zombienet_tests/functional/0012-elastic-scaling-mvp.zndsl

```diff
@@ -0,0 +1,27 @@
+Description: Test that a paraid acquiring multiple cores does not brick itself if ElasticScalingMVP feature is enabled
+Network: ./0012-elastic-scaling-mvp.toml
+Creds: config
+
+# Check authority status.
+validator: reports node_roles is 4
+
+validator: reports substrate_block_height{status="finalized"} is at least 10 within 100 seconds
+
+# Ensure parachain was able to make progress.
+validator: parachain 2000 block height is at least 10 within 200 seconds
+
+# Register the second core assigned to this parachain.
+alice: js-script ./0012-register-para.js return is 0 within 600 seconds
+
+validator: reports substrate_block_height{status="finalized"} is at least 35 within 100 seconds
+
+# Parachain will now be stalled
```

Comment:
Can we have a check for this actually happening? block height is at most?

Reply:
yes, I can add that.

The test should act as a regression test even without this check though. Before this PR, `parachain 2000 block height is at least 30 within 200 seconds` would have not been satisfied because the parachain would never get unstalled and it'd be stuck at a height around 15.

Reply:
done